JamesNojekChallenge/Database.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+def oldToNewDB(filename):
+    try:   
+        with open(filename, "r") as f:
+            text = json.load(f) #get data from file
+            print("Input the new database name: ")
+            DBname = str(input())   #select database to edit
+            for i in range (0, len(text['Customer Records'])):
+                #define database being edited and column names being edited
+                print("INSERT INTO " + DBname + " (Record ID, Name, Cell Phone, Work Phone, Email, Address, Basic Widget Order, Advanced Widget Order, Protection Plan)")

```

Comment:
I don't see anywhere in the code where you define the schema of the SQL DB. How does it work to insert into `DBName`?

Reply:
I've never used SQL, so I focused on the insert commands, not knowing about these other requirements. That's my mistake, but I'd like to think that in a team based environment I would've asked someone more knowledgeable when using a new language.